src/components/AbilitySkill.js

```diff
@@ -9,13 +11,13 @@ const AbilitySkill = ({ getAbilitiesLinks, getSkillsLinks }) => {
         <section className='main__section'>
           <h3 className='main__subheading'>Abilities</h3>
           <div className='main__links'>
-            {getAbilitiesLinks()}
+            {getLinks('abilities')}
           </div>
         </section>
```

Comment:
I see that these lines of code are repeated approximately 8 times between this file, `RaceClass`, `Gameplay` and `Glossary`

I would consider turning this into its own reusable component and have it accept a links prop and then you can invoke the function to get links and pass that in. 

For reference in my future comments i'll refer to this potential component as `<LinkSection />`

Reply:
Added to existing issue for directories